controllers/notification/mailgun_notification_delivery_service.go

```diff
@@ -114,7 +115,7 @@ func (s *MailgunNotificationDeliveryService) Send(notification *toolchainv1alpha
 		message.SetReplyTo(s.ReplyToEmail)
 	}
 
-	message.SetHtml(body)
+	message.SetHtml(html.EscapeString(body))
```

Comment:
but wouldn't this mean that the html fomat we have in our notification templates is not applied (since it's escaped)? I mean, we don't want to escape the body for all notifications, right? Only the body of the suspicious notifications created by the miner detector. we don't want to modify the expected behavior for the standard notifications sent to end-users.

Reply:
You're right, this would likely impact templates that contain html tags.  Perhaps the best way to approach this is to escape the string when the notification is actually created, rather than in the delivery service.  Where are the miner detector notifications created from?